webpack.config.js

```diff
@@ -49,6 +49,17 @@ module.exports = {
             }
           }
         ]
+      },
+      {
+        test: /\.css$/,
+        use: [
+          {
+            loader: 'style-loader'
+          },
+          {
+            loader: 'css-loader'
+          }
+        ]
```

Comment:
why do we need css loader?

Reply:
cause I need to take the class from shaka, I imported dist/controls.css

Reply:
the way you render it will use impoirt the entire css class to our project.
Why do we need it?

Reply:
cause shaka to create a container with shaka-text-container and we need this CSS class to render

Reply:
got ya - but as it takes all of the classes there and it is huge we would want to reduce it

Reply:
I can extract the relevant class but it won't change if shaka will change it